setup.cfg

```diff
@@ -1,96 +1,8 @@
 [metadata]
 name = aacgmv2
-version = 2.6.3
-long_description = file: README.rst, CHANGELOG.rst
-long_description_content_type = text/x-rst
-keywords = aacgm
-	   aacgm-v2
-           aacgmv2
-           magnetic coordinates
-           altitude adjusted corrected geomagnetic coordinates
-           mlt
-           magnetic local time
-           conversion
-           converting
-classifiers = 
-   Development Status :: 5 - Production/Stable
-   Intended Audience :: Science/Research
-   License :: OSI Approved :: MIT License
-   Operating System :: Unix
-   Operating System :: POSIX
-   Operating System :: MacOS :: MacOS X
-   Operating System :: Microsoft :: Windows
-   Programming Language :: Python
-   Programming Language :: Python :: 3
-   Programming Language :: Python :: 3.7
-   Programming Language :: Python :: 3.8
-   Programming Language :: Python :: 3.9
-   Programming Language :: Python :: 3.10
-   Programming Language :: Python :: Implementation :: CPython
-   Topic :: Software Development :: Libraries
-   Topic :: Scientific/Engineering :: Physics
-   Topic :: Utilities
-
-[options]
-zip_safe = False
-install_requires = numpy
-include_package_data = True
-include_entry_points = True
-
-[aliases]
-release = register clean --all sdist
+version = 2.7.0
```

Comment:
I think you could drop the duplicate metadata here since python 3.9 is the minimum supported version.

Reply:
#95